chunk/schema.go

```diff
@@ -561,37 +403,41 @@ func (v5Entries) GetWriteEntries(_, through uint32, tableName, hashKey string, l
 		encodedValueBytes := encodeBase64Value(value)
 		entries = append(entries, IndexEntry{
 			TableName:  tableName,
-			HashValue:  hashKey + ":" + string(key),
-			RangeValue: buildRangeKey(encodedThroughBytes, encodedValueBytes, chunkIDBytes, rangeKeyV4),
+			HashValue:  bucketHashKey + ":" + string(metricName) + ":" + string(key),
```

Comment:
Not sure how Go implements under the hood, but in Python or Java, sprintf would be more performant, as repeated string concatenation is quadratic.